### PR TITLE
dev-qt/qtcore: Pass -no-feature-statx to configure

### PR DESCRIPTION
**Pull request by a17r**
Either way, `/usr/include/qt5/QtCore/5.11.3/QtCore/private/qtcore-config_p.h` contains the same line:
``#define QT_FEATURE_statx -1``

Should it be passed in eclass instead?

### CODE REVIEW

**Comment by a17r** [comment]
Nevermind, I missed the fact this machine is currently stuck at sys-libs/glibc-2.26-r7

File: dev-qt/qtcore/qtcore-5.11.3-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+QT5_GENTOO_PRIVATE_CONFIG=(
+	!:network
+	!:sql
+	!:testlib
+	!:xml
+)
+
+src_configure() {
+	local myconf=(
+		-no-feature-statx
```
Comment (Pesa) [inline dev-qt/qtcore/qtcore-5.11.3-r1.ebuild]: Please add a comment next to or above this line mentioning the bug number

Othat than that, patch LGTM. I suppose we should do the same to 5.12.x in the overlay(?)

Comment (a17r) [reply dev-qt/qtcore/qtcore-5.11.3-r1.ebuild]: I guess we'll have to keep this line for at least as long as Gentoo supports <4.14 kernels.

Comment (Pesa) [reply dev-qt/qtcore/qtcore-5.11.3-r1.ebuild]: I'm afraid so, unless upstream fixes `statx` detection...

**Pull request merged**